scripts/shared/dbRequestUtils.js

```diff
@@ -88,7 +88,7 @@ gpii.dbRequest.configureStep = function (details, options) {
         );
     } else {
         request = gpii.dbRequest.queryDatabase(
-            details.requestUrl, response, details.requestErrMsg
+            details.requestUrl, response, details.requestErrMsg, togo
```

Comment:
Shall the similar error handling be added for `gpii.dbRequest.createPostRequest`?

Reply:
Yes!  How did I miss that?  Thanks for catching it.